tests/Controller/EnMarche/CommitteeControllerTest.php

```diff
@@ -285,6 +287,81 @@ public function testAuthenticatedCommitteeHostCanShowCommitteePage()
         $this->assertTrue($this->seeMessageForm($crawler));
     }
 
+    public function testNoEditLinkWithAnonymousUser()
+    {
+        $crawler = $this->client->request(Request::METHOD_GET, '/comites/en-marche-paris-8');
+
+        $this->assertResponseStatusCode(Response::HTTP_OK, $this->client->getResponse());
+
+        $result = $crawler->filter('.feed-rc-dropdown');
```

Comment:
It would be easier for everybody to use `selectLink`, `selectButton`, etc. `methods` when possible. It brings a lot more context to the test than css class.